phpBB/assets/javascript/installer.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Installer's AJAX frontend handler
+ */
+
+(function($) { // Avoid conflicts with other libraries
+    // Global variables
```

Comment:
I meant global as global within the wrapper function or whatever it is called. Any suggestion for naming that?